requirements.txt

```diff
@@ -4,6 +4,7 @@ requests~=2.26.0
 kubernetes~=18.20.0
 retry~=0.9.2
 waiting~=1.4.1
+assisted_service_client
```

Comment:
maybe we want to pin assisted_service_client version?
we query v1 API at the time being, so I don't think we can use the latest version yet

Reply:
As @michaellevy101 explained to me, we need here the most updated version of the client

Reply:
But I removed it because it already contained inside the image